compose/service.py

```diff
@@ -399,6 +401,12 @@ def _get_container_create_options(self, override_options, one_off=False):
             self.containers(stopped=True, one_off=one_off),
             one_off)
 
+        if 'labels' in container_options:
+            labels_dict = ['io.docker.compose:' + label for label in container_options['labels']]
```

Comment:
A motivation for a separate prefix for labels specified in the composition definition I can see is to be able to differ between the labels specified in the composition and the labels specified with `docker-compose run --label`.

This implements separation of concern between labels specified by the developer and whoever is running the composition in a particular environment. Otherwise `docker-compose run` invocations are required to always specify and overwrite all possible labels used in an environment if the guy running the workloads wants to avoid a developer mistakenly specifying a label used for operational concerns.

Simple example:
You are using a label `profile=production` which influences resource allocation and scheduling. You do not want compositions to inherit these privileges unless the label was specified by the operator invoking `docker-compose run`.

A richer write-up of this separation of concern can be found here:
https://github.com/docker/docker/issues/11187